spec/features/admin/schools_spec.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Schools', type: :request do
+  let(:admin_user) { create(:admin_user) }
+
+  before do
+    sign_in_as(admin_user)
+  end
+
+  describe 'GET #index' do
+    it 'responds 200' do
+      get '/admin/schools'
```

Comment:
Minor: It might be nicer to use `admin_schools_path` here...?

Reply:
Good spot! I've amended the commit.